src/modules/clearCompletedTask.js

```diff
@@ -0,0 +1,14 @@
+import createNewTask from './createNewTask.js';
+
+export default function clearCompletedTasks(tasks, todoList) {
+  tasks = tasks.filter((task) => !task.completed);
+  localStorage.setItem('tasks', JSON.stringify(tasks));
+  todoList.innerHTML = '';
+
```

Comment:
Good Job implementing this functionality, but it isn't working as expected. Kindly fix it 👍🏽
[Screencast from 19-08-2023 23:41:57.webm](https://github.com/tasnimchy96/todoList.github.io/assets/44624138/9e794f15-5bf6-40cf-a2fe-48a0a7588f1f)